source/faq.txt

```diff
@@ -97,7 +97,7 @@ applications:
 
 .. code-block:: csharp
    
-   var client = new MongoClient("<connection string>");
```

Comment:
S: I think `<connection string>` is the best placeholder, suggesting that you change the placeholder to this in all locations.